rueidisprob/bloomfilter.go

```diff
@@ -0,0 +1,344 @@
+package rueidisprob
+
+import (
+	"context"
+	"errors"
+	"math"
+	"strconv"
+
+	"github.com/redis/rueidis"
+)
+
+const (
+	// NOTE: https://redis.io/docs/data-types/bitmaps/
+	maxSize = 1 << 32
+)
+
+const (
+	addMultiScript = `
+local hashIterations = tonumber(ARGV[1])
+local numElements = tonumber(#ARGV) - 1
+local filterKey = KEYS[1]
+local counterKey = KEYS[2]
+
+local bitfieldArgs = { filterKey }
+for i=2, numElements+1 do
+    table.insert(bitfieldArgs, 'SET')
+    table.insert(bitfieldArgs, 'u1')
+    table.insert(bitfieldArgs, ARGV[i])
+    table.insert(bitfieldArgs, '1')
+end
+
+local setBits = redis.call('BITFIELD', unpack(bitfieldArgs))
+
+local counter = 0
+local oneBits = 0
+for i=1, #setBits do
+	if setBits[i] == 1 then
+		oneBits = oneBits + 1
+	end
```

Comment:
```suggestion
	oneBits = oneBits + setBits[i]
```
Could we just do this?

Reply:
[299d097](https://github.com/redis/rueidis/pull/493/commits/299d09754639f4f78bdb8c04906755a382c0467f)